src/OrleansRuntime/Catalog/GrainCreator.cs

```diff
@@ -32,7 +33,7 @@ public GrainCreator(IGrainRuntime grainRuntime, IServiceProvider services = null
         public Grain CreateGrainInstance(Type grainType, IGrainIdentity identity)
         {
             var grain = _services != null
-                ? (Grain) _services.GetService(grainType)
+                ? (Grain) _services.GetRequiredService(grainType)
                 : (Grain) Activator.CreateInstance(grainType);
```

Comment:
I don't think it's possible (at least not expected) to have a null service provider, right?